object.js

```diff
@@ -0,0 +1,56 @@
+let fastFurious = {
+  titel: "fast&furious",
+  year: 2001,
+  director: "Rob Cohen",
+  actors: "vin Diesel - paul walker ",
+  rating: 7,
+  timesWatched: 23,
+  recommend: function () {
+    console.log(`${this.titel} this move is amezing`);
+  },
+  watch: function () {
+    this.timesWatched = this.timesWatched + 1;
+  },
+  setRating: function (newRating) {
+    console.log((this.rating = newRating));
```

Comment:
You shouldn't put this inside a console log, if you wanted to display the new rating value inside this method, you should do it like this instead:
```
this.rating = newRating;
console.log(this.rating);
```